app/assets/javascripts/discourse/app/helpers/category-badge.js

```diff
@@ -1,12 +1,14 @@
 import { categoryLinkHTML } from "discourse/helpers/category-link";
-import { registerUnbound } from "discourse-common/lib/helpers";
 import { isPresent } from "@ember/utils";
+import { registerRawHelper } from "discourse-common/lib/helpers";
 
-registerUnbound("category-badge", function (cat, options) {
+registerRawHelper("category-badge", categoryBadge);
+
+export default function categoryBadge(cat, options = {}) {
   return categoryLinkHTML(cat, {
     hideParent: options.hideParent,
     allowUncategorized: options.allowUncategorized,
     categoryStyle: options.categoryStyle,
     link: isPresent(options.link) ? options.link : false,
   });
-});
+}
```

Comment:
This is necessary for gjs, or we can't import helpers only defined through `registerUnbound` (registerRawHelper is the new API to replace registerUnbinound), still, we now need to export a function to be usable in gjs.